kytos/core/auth.py

```diff
@@ -56,12 +182,18 @@ def __init__(self, controller):
             controller(kytos.core.controller): A Controller instance.
 
         """
+        self.user_controller = self.get_user_controller()
+        self.user_controller.bootstrap_indexes()
         self.controller = controller
-        self.namespace = "kytos.core.auth.users"
         self.token_expiration_minutes = self.get_token_expiration()
         if self.controller.options.create_superuser is True:
             self._create_superuser()
```

Comment:
@Alopalao, let's also adapt this method `self._create_superuser()` to not return the created user, and similarly to `_create_user(self)`, only return a string "User successfully created", that way the password isn't printed out when a network operator is calling this method in the CLI. Regarding the exceptions, please catch them accordingly and print the error, to avoid a traceback in the Kytos console. Since this method creating via Kytos console is what we have available to bootstrap users it's important that we have this. 

Once you adapt this, please make sure, to exercise the ``--create-superuser`` when starting `kytosd` as well.

Reply:
For the exceptions it was printing `The browser (or proxy) sent a request that this server could not understand.` I added messages for `ValidationError` would be `Inputs could not be validated.` and for `DuplicateKeyError` is `{username} already exist.`

Reply:
Sounds good. Let me close this thread.